Templates/Resources/Resources.resx

```diff
@@ -0,0 +1,276 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
There are a few bugs in the strings, please check this commit in my fork and apply the same,
https://github.com/soninaren/azure-webjobs-sdk-templates/commit/dd947cfb3a9db36ff4762f1ba5a4285728d4a860